backend/user/views.py

```diff
@@ -95,14 +95,15 @@ def post(self, request):
         title = request.data.get("title", None)
         body = request.data.get("body", None)
         delay = request.data.get("delay", 0)
+        is_dev = request.data.get("isDev", False)
 
         if None in [service, title, body]:
             return Response({"detail": "Missing required parameters."}, status=400)
         if service not in dict(NotificationSetting.SERVICE_OPTIONS):
             return Response({"detail": "Invalid service."}, status=400)
 
         success_users, failed_users = send_push_notifications(
-            usernames, service, title, body, delay
+            usernames, service, title, body, delay=delay, is_dev=is_dev
```

Comment:
Is there a reason why we defined the variables `delay=delay` and `is_dev=is_dev` here? Not sure if needed.